### ISSUE TITLE
Distribution Downloader does not show download size

### ISSUE THREAD

**Issue by hackerb9** [SevenBits/Mac-Linux-USB-Loader#71]
The automatic distribution downloader is a fabulous feature, but there's a small enhancement that I think would help a lot. In addition to "Distribution Name" and "Current Version", it would be nice to have a column for "Download Size". 

Currently, there is no way to guess how long a download might take until one starts the download. 

For example, since I am simply interested in trying out Mac-Linux-USB-Loader and the distribution installed does not matter. I wanted the smallest (quickest to download) one, but I can't tell what's smaller. Ubuntu? Elementary? Zorin? If the sizes were visible, one could guess how long the download was going to take before starting it.

**Comment by SevenBits** [comment]
@hackerb9 This sounds like a good idea; I'll look into it. 3.1 is feature-complete right now but in the following release I'll definitely try to include this feature. Thanks for sharing!
